BD-3799 Update Global Control Group membership (#8705)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/testing/global_control_group.md

## Export your control group members {#export-group-members}

If you'd like to see which users are in your Global Control Group, you can export your Group's members by CSV or API. 

To run a CSV export, navigate to the **Global Control Group Settings** tab and click <i class="fas fa-download"></i>&nbsp;**Export**. To export by API, use the [`/users/export/global_control_group` endpoint]({{site.baseurl}}/api/endpoints/export/user_data/post_users_global_control_group/).

{% alert important %}
Historical control groups are not preserved, so you can only export the members of your current group. Make sure to export any necessary information before disabling a control group.
{% endalert %}

## View whether a user is in a Global Control Group

> You can view Global Control Group membership by going to the **Engagement** tab of an individual user's profile and scrolling to the **Miscellaneous** section.

Reviewer: ```suggestion
You can view Global Control Group membership by going to the **Miscellaneous** section in the **Engagement** tab of an individual user's profile.
```
small suggestion, lgtm though so feel free to merge 🫡
